src/headless/tasks/inasafe_wrapper.py

```diff
@@ -290,7 +292,8 @@ def run_multi_exposure_analysis(
 def generate_report(
         impact_layer_uri,
         custom_report_template_uri=None,
-        custom_layer_order=None):
+        custom_layer_order=None,
+        custom_legend_layer=None):
```

Comment:
Do we have any analysis in inasafe-django that will use this features for now (custom_legend_layer)